### PR TITLE
Fix default keymaps for OLKB boards to play Startup Sound

### PR DESCRIPTION
**Pull request by drashna**
The dip_switch_update callback was overriding the default startup sound.  This should prevent that from happening, and still allow it to play sounds, or stop them, when appropriate.

Specifically, this was caused because the dip switch was only being called on change, but not on initialization.   The recent dip switch feature changes this so that it's always called, to properly set the initial state.  However, this was effectively disabling the startup sound. 

## Types of Changes
- [x] Bugfix
- [x] Keymap/layout/userspace (addition or update)

## Checklist
- [x] My code follows the code style of this project.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/preonic/keymaps/default/keymap.c [hunk 1]
```diff
@@ -260,40 +260,43 @@ void encoder_update_user(uint8_t index, bool clockwise) {
...
+                layer_off(_ADJUST);
+            }
+            break;
+        case 1:
+            if (active) {
+                muse_mode = true;
+            } else {
+                muse_mode = false;
+            }
+    xs}
```
Comment (fauxpark) [inline keyboards/preonic/keymaps/default/keymap.c]: @drashna seems you missed this 😉

Comment (drashna) [reply keyboards/preonic/keymaps/default/keymap.c]: Oh no :(

And why did Travis not catch that, either?

Comment (fauxpark) [reply keyboards/preonic/keymaps/default/keymap.c]:
```
Making all keymaps for preonic
QMK Firmware 0.7.13
make: *** No rule to make target 'preonic:all'. Stop.
|
|  QMK's make format recently changed to use folder locations and colons:
|     make project_folder:keymap[:target]
|  Examples:
|     make planck/rev4:default:dfu
|     make planck:default
|
The command "bash util/travis_build.sh" exited with 0.
```

😕

Comment (drashna) [reply keyboards/preonic/keymaps/default/keymap.c]: that ... would be why.  Opened another PR to fix that, and a couple of other issues that snuck by

**Pull request merged**